src/features/classification/security-controls/messages.js

```diff
@@ -0,0 +1,177 @@
+            'Short summary displayed for classification when both sharing and app download restrictions are applied to it',
+        id: 'boxui.securityControls.shortSharingApp',
+    },
+    shortDownloadApp: {
+        defaultMessage: 'Download and app restrictions apply',
+        description:
+            'Short summary displayed for classification when both download and app download restrictions are applied to it',
+        id: 'boxui.securityControls.shortDownloadApp',
+    },
+    shortAllRestrictions: {
+        defaultMessage: 'Sharing, download and app and restrictions apply',
+        description:
+            'Short summary displayed for classification when sharing, download and app download restrictions are applied to it',
+        id: 'boxui.securityControls.shortAllRestrictions',
+    },
+    sharingCollabOnly: {
+        defaultMessage: 'Shared links cannot be made publicly accessible.',
+        description: 'Bullet point that summarizes shared link restriction applied to classification',
+        id: 'boxui.securityControls.sharingCollabOnly',
+    },
+    sharingCollabAndCompanyOnly: {
+        defaultMessage: 'Shared links allowed for collaborators only.',
+        description: 'Bullet point that summarizes collaborators shared link restriction applied to classification',
+        id: 'boxui.securityControls.sharingCollabAndCompanyOnly',
+    },
+    externalCollabBlock: {
+        defaultMessage: 'External collaboration restricted.',
+        description:
+            'Bullet point that summarizes external collaboration blocked restriction applied to classification',
+        id: 'boxui.securityControls.externalCollabBlock',
+    },
+    externalCollabDomainList: {
+        defaultMessage: 'External collaboration limited to approved domains.',
+        description: 'Bullet point that summarizes external collaboration restriction applied to classification',
+        id: 'boxui.securityControls.externalCollabDomainList',
+    },
+    appDownloadBlock: {
+        defaultMessage: 'Some application restrictions apply.',
+        description: 'Bullet point that summarizes application download blocked restriction applied to classification',
+        id: 'boxui.securityControls.appDownloadBlock',
+    },
+    appDownloadList: {
+        defaultMessage: 'Some applications will be restricted; {appNames}',
+        description: 'Bullet point that summarizes application download restriction applied to classification',
+        id: 'boxui.securityControls.appDownloadList',
+    },
+    appDownloadListOverflow: {
+        defaultMessage: 'Only select applications are allowed; {appNames} +{remainingAppCount} more',
+        description:
+            'Bullet point that summarizes application download restriction applied to classification. This variation is used when the list of applications is longer than the configured threshold',
+        id: 'boxui.securityControls.appDownloadListOverflow',
+    },
+    // Web Download Restrictions
```

Comment:
Confirmed that it could potentially cause translation issues to translate the platform (e.g. web, desktop, mobile) string separately and pass it as a parameter, hence the need to duplicate these for all three platforms